src/scss/partials/_block.scss

```diff
@@ -48,6 +48,11 @@
       text-decoration: none;
     }
 
+    .give-help__contribute a {
```

Comment:
you shouldn't have raw html tags in your css, you need to give the a tag a class too

Reply:
Cheers, will amend